atomone-1/README.md

```diff
@@ -0,0 +1,81 @@
+# 🔗 `atomone-1`
+
+![chain-id](https://img.shields.io/badge/chain%20id-atomone--1-blue?style=for-the-badge)
+![version: v1.0.0](https://img.shields.io/badge/version-v1.0.0-green?style=for-the-badge)
+<!-- ![genesis-time](https://img.shields.io/badge/%E2%8F%B0%20genesis%20time-2024--02--27T13%3A00%3A00Z-red?style=for-the-badge) -->
+
+
+## Register in the Genesis
+
+To register your validator node in the `genesis.json` you just need to provide a signed `gentx` with an initial delegation of `1000000uatone`.
+
+```sh
+# Init node
+atomeoned init your-moniker --chain-id atomone-1
+
+# Create keys, be careful with the mnemonic 👀
+atomeoned keys add your-key-name
```

Comment:
Should we document the import of cosmos-hub keys ?

Reply:
I don't believe we should encourage people to use the same validators key as the hub. If people want to do it, they can. But that's not a best practice.

Reply:
I don't think sharing the same key is a good practice or even using the same as a testnet, it's good to have a fresh key
Maybe adding how to use different tool like tkms or horcrux could be good

Reply:
Love the "tmkms" or "horcrux" tutorial idea, It could be in a different PR, let's just have this quickly merge so we have material to contribute to

Reply:
Sounds good guys, thx for the feedback, I close this comment.